.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+**List the core changes of this PR**
+
+<!--
+* Adds a new mapping based on circular stochastic lists by Penguin et al.
+* Fixes a memory leak in com::LeakyCommunicator
+* Removes obsolete function Vertex::isQuestionable()
+* Refactors checks of the Mesh class
+* Changes algorithm used in Triangle::computeNormal()
+-->
+
+**Motivation**
+
+<!--
+Short rational why preCICE need this change.
+-->
+
+**Additional Information**
+
+<!--
+Links to other issues/PRs and resources:
+
+Closes #123
+Related to #123
+Based on PR #123
+See [FindBoost](https://cmake.org/cmake/help/latest/module/FindBoost.html)
+-->
+
+**TODO**
+
+* [ ] I added a changelog file with this PR number in `docs/changelog/` if there are noteworthy changes.
+* [ ] I ran `tools/formatting/check-format` and everything is formatted correctly.
+* [ ] I sticked to C++11 features
+* [ ] I sticked to CMake version 3.10
```

Comment:
Those things our tests should reveal, right? (maybe not right now but soon)
Not sure if we need these items.

Reply:
I really like these checkboxes and I would like to keep them. Even at the stage of a draft Pull Request, the contributor can think of these early enough. Our tools should check these, if possible, but this way we avoid surprises for the contributor.

Reply:
My intention is to make the contributor aware of the requirement when opening the PR. 
This way the contributor has to actively say "yes this is ok".